docs/contributing/test-components-using-accessibility-acceptance-criteria.md

```diff
@@ -0,0 +1,154 @@
+
+## How the GOV.UK Design System's accessibility acceptance criteria relate to WCAG
+
+The GOV.UK Design System accessibility acceptance criteria are a combination of the [WCAG (Web Content Accessibility Guidelines)](https://www.w3.org/TR/WCAG21/) and the general accessibility requirements that components need to meet.
+
+The GOV.UK Design System’s accessibility acceptance criteria are complementary to WCAG and do not replace it.
+
+## How to test components using accessibility acceptance criteria
+
+You need to [make sure your services work with a set of assistive technology and browser combinations](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies).
+
+We recommend you test your component by putting it inside a realistic page. For example, a complete page that uses realistic page content and contains elements like a header, footer and so on.
+
+Check if the desired behaviour happens when testing. For example, whether [screen readers announce the headings](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#how-to-test). As you test, take notes about whether the component or iteration met the criteria, or if there were issues you need to investigate further.
+
+We have created a [template for capturing findings from accessibility tests](https://docs.google.com/spreadsheets/d/1O0rWoH3ah_rj2iJ2We9VfyHNOr0gOVItXRy3vKzuYSw/edit). If it could help you, download it or make your own copy. If you cannot access the spreadsheet, [contact us]( https://design-system.service.gov.uk/#support) and we will make sure you get a copy.
+
+## GOV.UK Design System - accessibility acceptance criteria
+
+Before we publish a component on the GOV.UK Design System, it should meet the following accessibility requirements.
+
+### General accessibility acceptance criteria
+
+To show clearly to users, components should:
+
+- have text with a colour contrast of at least 4.5:1 between text and the background
+- not depend on colour alone to communicate information
+- not move (for example, flash or blink)
+- be visually distinct from other elements on the page
+- use [semantic markup](https://www.w3.org/WAI/WCAG21/Techniques/general/G115.html)
+- if necessary, hide and display content consistently for both screen reader users and sighted users - [learn more about hiding and displaying content](https://webaim.org/techniques/css/invisiblecontent/#intro)
+
+To make sure users can use them, components should at least be operable with:
+
+- a keyboard
+- voice
+- touch
+- a mouse
+- the rotor or equivalent, if your screen reader has one - [learn how to use the rotor and local and global context menus](https://dequeuniversity.com/screenreaders/)
+a screen reader in browse or reader mode
+- any other modes your screen reader might have - [learn how to use different modes](https://dequeuniversity.com/screenreaders/)
```

Comment:
Just wanted to check - are we intentionally linking to https://dequeuniversity.com/screenreaders/ twice?

Reply:
Yeah, that's intentional. Bit clunky, I know. Our rationale was, including it twice hopefully makes it less likely users will pass over accessing content that could help them.